catalog-info.yaml

```diff
@@ -1,3 +1,18 @@
+# Declare a Backstage Component that represents your application.
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: elastic-package
+  description: elastic-package - Command line tool for developing Elastic Integrations
+
+spec:
+  type: tool
+  owner: group:ingest-fp # Find your group in https://backstage.elastic.dev/catalog?filters%5Bkind%5D=group
+  system: platform-ingest # Find your group in https://backstage.elastic.dev/catalog?filters%5Bkind%5D=system
```

Comment:
This is not yet shown in the list of services in backstage, otherwise LGTM

Reply:
This `platform-ingest` already appears in the list :+1: